lib/iris/analysis/cartography.py

```diff
@@ -500,8 +501,20 @@ def project(cube, target_proj, nx=None, ny=None):
             covering the globe.
 
     Returns:
-        An instance of :class:`iris.cube.Cube` and a list describing the
-        extent of the projection.
+        An tuple containing an instance of :class:`iris.cube.Cube` and a list
+        describing the extent of the projection.
```

Comment:
It'd be helpful to describe the structure of the extent, i.e. `[x_min, x_max, y_min, y_max]`.